aisdc/attacks/worst_case_attack.py

```diff
@@ -44,8 +44,8 @@ def __init__(self, **kwargs):
         self.__dict__["test_prop"] = 0.3
         self.__dict__["n_rows_in"] = 1000
         self.__dict__["n_rows_out"] = 1000
-        self.__dict__["in_sample_filename"] = None
-        self.__dict__["out_sample_filename"] = None
+        self.__dict__["training_preds_file"] = None
```

Comment:
please change the key to make it clear this is a file _name_

Reply:
Updated the names to "training_preds_filename" and "test_preds_filename".